zulipterminal/model.py

```diff
@@ -698,10 +698,12 @@ def get_stream_by_id(streams: List[List[Any]], stream_id: int
                     sort_streams(self.pinned_streams)
                     self.controller.view.left_panel.update_structure()
                     self.controller.update_screen()
-        elif event['op'] == 'peer_add':
+        elif(event['op'] == 'peer_add'
+             and event['stream_id'] in self.stream_dict):
             subscribers = self.stream_dict[event['stream_id']]['subscribers']
             subscribers.append(event['user_id'])
-        elif event['op'] == 'peer_remove':
+        elif(event['op'] == 'peer_remove'
+             and event['stream_id'] in self.stream_dict):
             subscribers = self.stream_dict[event['stream_id']]['subscribers']
             subscribers.remove(event['user_id'])
```

Comment:
I chose not to create a new variable for `event[stream_id]` because I'm not sure if all events will contain `stream_id`

Reply:
Do you mean refactoring it above the if/elif chain?

Could we add spaces in there after the keywords again?

Reply:
Yes, I meant making a new variable outside the if/elif block.

Isn't there a rule available for the spaces? I couldn't find any